docs/dev/frontend.rst

```diff
@@ -209,3 +209,7 @@ We distinguish development dependencies from runtime dependencies, and these sho
 Note that we currently don't have a way of mapping dependencies to plugins - dependencies are installed globally.
 
 To assist in tracking the source of bloat in our codebase, the command ``npm run bundle-stats`` is available to give a full readout of the size that uglified packages take up in the final Javascript code.
+
+Individual plugins can also have their own package.json for their own dependencies. Running ``npm install`` will also install all the dependencies for each activated plugin (inside a node_modules folder inside the plugin itself). These dependencies will only be available to that plugin at build time.
```

Comment:
My main reason for doing this, is in case different plugins depend on different versions of a particular node module, they can then use that particular version by specifying in their package.json, without having specify the version in their require statement.